spring-cloud-gcp-data-datastore/src/test/java/org/springframework/cloud/gcp/data/datastore/core/convert/TestDatastoreItemUnsupportedFields.java

```diff
@@ -16,15 +16,16 @@
 
 package org.springframework.cloud.gcp.data.datastore.core.convert;
 
-import java.math.BigInteger;
+
+import java.util.Objects;
 
 /**
  * @author Dmitry Solomakha
  */
 class TestDatastoreItemUnsupportedFields {
```

Comment:
See if you can add a field here that is of type `Object` and uses a converter to convert to `String`. Something like where your Object is actually a string and you store it in reverse.